app/src/main/java/org/openlake/sampoorna/data/constants/Constants.kt

```diff
@@ -25,7 +25,8 @@ object Constants {
      * Other useful constants
      */
     val months = listOf("January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December")
```

Comment:
When dealing with constant cases, you can come up with enums instead of strings.

Reply:
I see, but I made this list for displaying month names so maybe enums won't be applicable here 🤔

Reply:
You can do something like this:
` enum class Months(val month: String){
JANUARY("January")
}
`
and access the value by: `Months.JANUARY.month`

Reply:
It's just an example though, you will face similar conditions later a lot. I learned it after like 10 Code Reviews kek. So yeah, a good practice to do so.

Reply:
Ah I see 😮